### PR TITLE
Added TIC Game Engine, SunVox, And Alphabetized Music Section

### PR DESCRIPTION
**Pull request by StuffBySpencer**
Added the TIC - Tiny Computer Game Engine

### CODE REVIEW

**Comment by StuffBySpencer** [comment]
Oh hey, by the way, I'm new to doing PR on GitHub, so I was hoping I could get some feedback.

When I make multiple changes [ _like in this PR, for example_ ], should I just submit them all in one single PR and keep updating the commits as I go along [ _like I did here_ ]?
Or is it better to fork the base repo again and do a brand new PR for each and every addition / change?

Would love to know which is a better way to go about it, thanks!

**Comment by Kavex** [comment]
Well in layman's terms, Make sure your fork is up to date, do your changes to your fork, double check it, then do a pull request to sync it with the original repo. Basically do all the changess you want first the pull request. The github program makes it easy to update the fork if you are new.

**Comment by StuffBySpencer** [comment]
Ah, ok, thank you very much for the help! :)

**Comment by ellisonleao** [comment]
Thanks @StuffBySpencer and @Kavex
